pcp/columns/cmd

```diff
@@ -0,0 +1,9 @@
+# pcp-htop(1) configuration file - see pcp-htop(5)
```

Comment:
We shouldn't provide dynamic column configs that mirror static columns.  Remove this file & I'll send through some other realistic samples we should use shortly.